move.c

```diff
@@ -15,6 +14,11 @@ float current_cell_shift(void)
 	       MICROMETERS_PER_METER;
 }
 
+/**
+ * @brief Initialize mouse position
+ *
+ *  Assume the mouse tail is initially touching a wall.
```

Comment:
Minor stuff:

- Ending `.` in the subject
- Double space before "Assume"
- I would say "Assumes" instead of "Assume"

Reply:
41ef97a5c

Reply:
41ef97a5c